### DISCUSSION TITLE
Controller not working properly

### DISCUSSION THREAD

**Discussion by ji2k** [react-hook-form/react-hook-form#5837] [Q&A] [answered]
When I use `Controller` or `useController`, I only pass `field.value` and `field.onChange` into `input` component, then I cannot use `setValue`. but when I call `getValues` function, I still get the value i just passed in.

[Codesandbox link](https://codesandbox.io/s/gracious-sunset-q5dy4)

- Device: [dell]
- OS: [window 10]
- Browser [chrome]
- Version [7.10.0]

**Comment by bluebill1049** [comment, marked as answer]
I am looking into this, if you create an issue, please follow our template.

**Reply by bluebill1049** [reply]
fixed and released the patch.

**Reply by bluebill1049** [reply]
https://github.com/react-hook-form/react-hook-form/issues/5838
